src/simulations/lava-coder/vog-simulation.ts

```diff
@@ -222,4 +231,11 @@ export class VogSimulation {
 
     this.sendUpdateMessage(true);
   }
+
+  public setPhase(phase: "creation" | "dispersion") {
+    this.phase = phase;
+    if (phase === "dispersion") {
+      this.vogPulses = this.vogPulses / 2;
```

Comment:
A comment here would be useful. It made sense to me because of the comment in the PR description: "vog will continue to disperse for half as many steps," but it would be useful to say that in the code.